scripts/ci/in_container/entrypoint_ci.sh

```diff
@@ -143,53 +142,31 @@ if [[ ${INTEGRATION_KERBEROS:="false"} == "true" ]]; then
 fi
 
 
-if [[ "${RUNTIME}" == "" ]]; then
-    # Start MiniCluster
-    java -cp "/opt/minicluster-1.1-SNAPSHOT/*" com.ing.minicluster.MiniCluster \
-        >"${AIRFLOW_HOME}/logs/minicluster.log" 2>&1 &
+# Start MiniCluster
+java -cp "/opt/minicluster-1.1-SNAPSHOT/*" com.ing.minicluster.MiniCluster \
```

Comment:
are you always starting minicluster as now the assumption is no one would run the k8s tests in the image?

Reply:
No. Minicluster has always been started in CI image. This is something we have to get rid of. It was not started when kubernetes runtime was used but now we completely got rid of kubernetes runtime in CI image and we do not use CI image in kubernetes tests a all so I removed the "if kubernetes" simply. 

There is a separate issue to get rid of minicluster from the CI image #8784